packages/zoltan2/test/connectivity/meshData/README.txt

```diff
@@ -0,0 +1,42 @@
+Status:  8/8/18
+
+File formats:
+
+*.quad.msh -- mesh files with quad elements
+line 1:  two meaningless words
+line 2:  #meshvertices #elements #floatingBorderEdges (<== always zero)
+lines 3-#meshvertices+2:  mesh vertex numbers (one based)
+remaining lines:  mesh vertices contained within each element
+          four vertices and meaningless flag (<== always zero)
+
+*-basal-friction.ascii
+line 1:  #meshvertices
+remaining lines:  1 if vertex is grounded; 0 otherwise
```

Comment:
ant*km-basal-friction.ascii contains floating point numbers.
Are the files wrong?  Or is this documentation wrong?

Reply:
The documentation is correct for files created by hand. I will add a note that real inputs have floating point numbers for the basal friction.